core/chain/coin/price/solana/getSolanaTokenUSDValue.ts

```
@@ -0,0 +1,63 @@
+const generateSolanaTokenQuoteUrl = (
+  inputMint: string,
+  outputMint: string,
+  amount: string,
+  slippageBps: string
+): string => {
+  return `https://quote-api.jup.ag/v6/quote?inputMint=${inputMint}&outputMint=${outputMint}&amount=${amount}&slippageBps=${slippageBps}`;
+};
+
+export const getSolanaTokenPrices = async ({
+  coins,
+  fiatCurrency = defaultFiatCurrency,
```

ESLint

[error] 'fiatCurrency' is assigned a value but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)